control-plane/cni/main.go

```diff
@@ -22,36 +26,58 @@ import (
 )
 
 const (
-	keyCNIStatus    = "consul.hashicorp.com/cni-status"
-	keyInjectStatus = "consul.hashicorp.com/connect-inject-status"
-	injected        = "injected"
+	// These annotations are duplicated from control-plane/connect-inject/annotations.go in
+	// order to prevent pulling in dependencies.
```

Comment:
You meant control-plane/config?  I am not sure if you have the same problem of pulling in all the dependencies or not.

Reply:
By pulling in dependencies, do you mean importing the package?

I was thinking if there was a common package with all of the annotations, we could import it and use `annotation.NameOfAnnotation`. That way we prevent duplication that could lead to bugs.

Reply:
Yes, by pulling in the package and adding a bunch of package to go.mod/sum. The annotations package would need to sit outside of control-plane to prevent that.  I've seen Istio do a similar thing to what you are proposing and it is pretty nice... `annotation.Foo` is nice to read IMHO.

I originally had the annotations down in control-plane/connect-inject/annotations.go but the cni plugin is an independent binary even though it is under control-plane/cni.  It pulled in A LOT of packages that it just doesn't need (like consul sdk/api). Hence, the duplication.

Reply:
I understand now! I didn't realize it was an independent binary.